facturedata/core.py

```diff
@@ -3,13 +3,45 @@
 import collections
 import json
 import sys
+from abc import ABCMeta, abstractmethod
 
 ordered_dict_version = (3, 6)
 HAS_DEFAULT_ORDERED_DICT = sys.version_info > ordered_dict_version
 
 #############################################################################
 
 
+class FactureRefObj(metaclass=ABCMeta):
```

Comment:
Looks like we're not actually inheriting from this abstract base class, right?

Reply:
True, I'll toss out the metaclass